### PR TITLE
add bts spin app to spinup hub

### PR DESCRIPTION
**Pull request by rajatjindal**

### CODE REVIEW

**Review by VamshiReddy02 — APPROVED** [review]

File: content/api/hub/sample_bts_golang.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "web", "nuxt", "ssg"]
+
+[extra]
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Golang"
```
Comment (mikkelhegn) [inline content/api/hub/sample_bts_golang.md:11]:
```suggestion
language = "Go"
```

Has to be 'Go', to match the tag filters

**Review by mikkelhegn — CHANGES_REQUESTED** [review]

**Review by mikkelhegn — APPROVED** [review]

File: content/api/hub/sample_bts_golang.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title = "Behind the Scenes sample"
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "web", "nuxt", "ssg"]
```
Comment (itowlson) [inline content/api/hub/sample_bts_golang.md:5]:
```suggestion
tags = ["go", "web", "nuxt", "ssg"]
```

File: content/api/hub/sample_bts_golang.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Go"
+created_at = "2024-01-15T00:22:56Z"
+last_updated = "2024-01-15T00:22:56Z"
+spin_version = ">v1.5"
+summary =  "A fun social photo app built using Golang/Nuxt/Spin"
+url = "https://github.com/rajatjindal/behind-the-scenes"
+keywords = "slack, golang, nuxt.js, webhook"
```
Comment (itowlson) [inline content/api/hub/sample_bts_golang.md:17]: Feels like `slack` is interesting enough to merit a tag?  But not sure what keywords vs tags are for...

Comment (karthik2804) [reply content/api/hub/sample_bts_golang.md:17]: keywords are not displayed as tags but help with search. Tags are shown on the card.

File: content/api/hub/sample_bts_golang.md (hunk 2 above)
Comment (rajatjindal) [reply content/api/hub/sample_bts_golang.md:5]: done

File: content/api/hub/sample_bts_golang.md (hunk 3 above)
Comment (rajatjindal) [reply content/api/hub/sample_bts_golang.md]: I've added slack to tags

**Pull request merged**